cockpit/src/subscriptions-view.jsx

```diff
@@ -230,16 +206,52 @@ var SubscriptionsPage = React.createClass({
         );
     },
     renderSubscriptions: function() {
-        var entries = this.props.products.map(function(itm) {
-            var tabRenderers = [
-                {
-                    name: _("Product's Subscription Details"),
-                    renderer: SubscriptionProductDetails,
-                    data: itm,
-                },
-            ];
-            var columns = [ { name: itm.productName, 'header': true } ];
-            return <ListingRow columns={columns} tabRenderers={tabRenderers}/>;
+        var entries = this.props.products.map(function (itm) {
+            let icon_name;
+            let status_text;
+
+            if (itm.status === 'subscribed') {
+                icon_name = "fa pficon-plugged";
+            } else {
+                icon_name = "fa pficon-unplugged";
```

Comment:
I used these two icons too :-) in one of my WIP version.

These two icons has wrong vertical alignment, but I guess it can be fixed in our css file.